contracts/MechMarketplace.sol

```diff
@@ -60,11 +60,11 @@ contract MechMarketplace is IErrorsMarketplace {
     event SetMechFactoryStatuses(address[] mechFactories, bool[] statuses);
     event SetPaymentTypeBalanceTrackers(bytes32[] paymentTypes, address[] balanceTrackers);
     event MarketplaceRequest(address indexed priorityMech, address indexed requester, uint256 numRequests,
-        bytes32[] requestIds);
-    event MarketplaceDelivery(address indexed deliveryMech, address[] indexed requesters, uint256 numDeliveries,
+        bytes32[] requestIds, bytes[] requestDatas);
+    event MarketplaceDelivery(address indexed deliveryMech, address[] requesters, uint256 numDeliveries,
         bytes32[] requestIds, bool[] deliveredRequests);
     event Deliver(address indexed mech, address indexed mechServiceMultisig, bytes32 requestId, uint256 deliveryRate,
-        bytes data);
+        bytes requestData, bytes deliveryData);
```

Comment:
`Deliver` event for delivery with signatures: adding request and delivery data-s